editor/editor_inspector.cpp

```diff
@@ -2581,6 +2726,28 @@ void EditorInspector::_parse_added_editors(VBoxContainer *current_vbox, EditorIn
 	ped->added_editors.clear();
 }
 
+bool EditorInspector::_is_class_disabled_by_feature_profile() {
+	Ref<EditorFeatureProfile> profile = EditorFeatureProfileManager::get_singleton()->get_current_profile();
+	if (profile.is_null()) {
+		return false;
+	}
+
+	StringName class_name = object->get_class();
+
+	if (profile->is_class_disabled(class_name)) {
+		List<StringName> feature_chain;
+
+		feature_chain.push_back(class_name);
+
+		disabled_features_dialog->add_disabled_feature(feature_chain);
+
+		//won't see properties of a disabled class
```

Comment:
```suggestion
		// Won't see properties of a disabled class.
```
[Comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide) nitpick